webpack-loader.js

```diff
@@ -32,5 +37,21 @@ module.exports = function () {
 
     this.cacheable();
 
-    return util.format("module.exports = %s;", JSON.stringify(config));
+    var query = loaderUtils.parseQuery(this.query);
+
+    if (! isArray(query.whitelist)) {
+        throw new Error('No config property whitelist was specified! ' +
+            'To use misty-config as a webpack loader you must specify a whitelist as a query parameter.');
```

Comment:
Suggest error message indicates that whitelist is an array: `must specify a whitelist array as a query parameter`.